relay-server/src/services/processor.rs

```diff
@@ -2551,6 +2551,8 @@ impl EnvelopeProcessorService {
         limits: &[CardinalityLimit],
         buckets: Vec<Bucket>,
     ) -> Vec<Bucket> {
+        use relay_log::sentry::User;
```

Comment:
I'd import at the top or inline it where used.

Reply:
Frigging auto-import.